### PR TITLE
Next13 add basic routes and homepage (#991)

### LINKED ISSUE #991: [Next13 starter kit with CSS modules] - Add routing and create homepage
# Background
We want to demonstrate basic routing as part of each kit. To do so, we want a homepage that directs to all the examples we've created thus far.

# Acceptance
- [ ] Add and configure routing
- [ ] Create a basic homepage that lives at `/`
- [ ] Route the counter component to `/counter`
- [ ] Route the API component to `/api-example`

# Note:
- The mock was taken from a different kit. Change the text to be align with the current kit

# Mock
image

### PR DESCRIPTION
**Pull request by BigAB**
## Type of change
Add basic routes to next13-zustand-bulma starter

## Summary of change
- [x] Add and configure routing
- [x] Create a basic homepage that lives at /
- [x] Route the counter component to /counter
- [x] Route the API component to /api-example

## Checklist

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] This starter kit has been approved by the maintainers
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #991 
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: starters/next13-zustand-bulma/src/app/page.tsx [hunk 1]
```diff
@@ -1,91 +1,22 @@
...
-          <h2 className={inter.className}>
-            Deploy <span>-&gt;</span>
-          </h2>
-          <p className={inter.className}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
+      <header className={styles.header}>
+        <h1 className={styles.heading}>Create Next app with React Query and Tailwind</h1>
```
Comment (jdwilkin4) [inline starters/next13-zustand-bulma/src/app/page.tsx:10]: We should change the name here

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
tested it out locally and it looks good. 
Left one small comment. 

Also, I wasn't sure what we wanted to do with the file names for the counter and fetch components. It looks like a lot of the other starter kits are using this convention. 


Screenshot 2023-03-10 at 1 45 30 PM

File: starters/next13-zustand-bulma/src/app/page.tsx (hunk 1 above)
Comment (BigAB) [reply starters/next13-zustand-bulma/src/app/page.tsx:10]: > We should change the name here
Yes. Will do.

> Also, I wasn't sure what we wanted to do with the file names for the counter and fetch components. It looks like a lot of the other starter kits are using this convention.
> 
> Screenshot 2023-03-10 at 1 45 30 PM

This is actually the route of the `/counter` page, not the `Counter` component itself.

We can definitely do that "modlet" thing in the components, but in the `App` directory, the directories represent routes, and by putting a `page.tsx` in there we are basically opening up that route for an available HTML serving page.

So yes, but not here.

Comment (BigAB) [reply starters/next13-zustand-bulma/src/app/page.tsx]: Fixed title in [b745395](https://github.com/thisdot/starter.dev/pull/1169/commits/b7453959b1172227ec05be72437fba30371df97a)

**Review by jdwilkin4 — APPROVED** [review]
Looks good 👍

**Pull request merged**